core/blockchain.go

```diff
@@ -1613,7 +1613,7 @@ func (bc *BlockChain) insertChain(chain types.Blocks, verifySeals, setHead bool)
 		}
 		// perform the verkle fork if this is the fork block
 		if block.NumberU64() == 230081 {
-			proot := common.HexToHash("0x57c783c15b3e79dc2eb0d6a5d7d780936a2da4ce8ef842d64344c65dcd3a986e")
+			proot := common.HexToHash("0x28e53289db6d24182b4c39e7f325364eec2110332d8fa4e4e0d1983db2c58a24")
```

Comment:
This is the state root of the converted tree. This hasn't changed because of the optimizations or serialization, but it's just a discrepancy with the base [verkle/replay-change-with-tree-group-tryupdate](https://github.com/gballet/go-ethereum/tree/verkle/replay-change-with-tree-group-tryupdate) branch.

Something to note here is that [verkle/replay-change-with-tree-group-tryupdate](https://github.com/gballet/go-ethereum/tree/verkle/replay-change-with-tree-group-tryupdate) isn't using the latest `master` of go-verkle but some tentative PR.

See how the [go.mod of verkle/replay-change-with-tree-group-tryupdate](https://github.com/gballet/go-ethereum/blob/verkle/replay-change-with-tree-group-tryupdate/go.mod#L26) is using a commit that corresponds to [this unmerged PR](https://github.com/gballet/go-verkle/pull/322) instead of `go-verkle@master` (or at least, _some_ commit in the master history).

Probably this was getting ahead of trying some optimizations; but just mentioning it since this new `go-verkle` optimization is built on top of `master`, not that tentative PR.

This is a separate topic by itself and may require some investigation... probably worth another chat.